### PR TITLE
Support feature unhashing when explaining regression predictions

### PR DESCRIPTION
**Pull request by lopuhin**
Adding a small helper that handles unhashing and coef_scale.

Test will need a small tweak to work with #21, I'll do it here or in #21.

### CODE REVIEW

File: eli5/sklearn/explain_prediction.py [hunk 1]
```diff
@@ -143,9 +149,10 @@ def _get_X(doc, vec=None, vectorized=False):
 @explain_prediction.register(SGDRegressor)
 def explain_prediction_linear_regressor(
         clf, doc, vec=None, top=_TOP, target_names=None,
-        feature_names=None, vectorized=False):
+        feature_names=None, vectorized=False, coef_scale=None):
     """ Explain prediction of a linear regressor. """
-    feature_names = get_feature_names(clf, vec, feature_names=feature_names)
+    vec, feature_names, coef_scale = _handle_vec(
```
Comment (kmike) [inline eli5/sklearn/explain_prediction.py]: `coef_scale` variable is unused (see `_weights` function);

Comment (lopuhin) [inline eli5/sklearn/explain_prediction.py]: Right, oh my! Thanks for the catch @kmike, fixed now.
But the score looks suspicious for some examples, I want to check if there could be something else here.

**Comment by kmike** [comment]
Thanks @lopuhin!
